src/plugins/intel_gpu/src/plugin/plugin.cpp

```diff
@@ -655,7 +655,7 @@ uint32_t Plugin::get_max_batch_size(const ov::AnyMap& options) const {
     uint32_t n_streams = static_cast<uint32_t>(config.get_num_streams());
     uint64_t occupied_device_mem = 0;
     auto statistic_result = get_metric(ov::intel_gpu::memory_statistics.name(), options).as<std::map<std::string, uint64_t>>();
-    auto occupied_usm_dev = statistic_result.find("usm_device_current");
+    auto occupied_usm_dev = statistic_result.find("usm_device");
```

Comment:
* could you add a test for this?
* could you refactor the code a bit to avoid using string literal?